docs/config.toml

```diff
@@ -158,6 +158,11 @@ relativeURLs = "true"
     weight = 70
     parent = "scripting"
     [[menu.index]]
+    name = "Metrics"
+    url = "/topics/metrics/"
+    weight = 74
+    parent = "topics"
+    [[menu.index]]
```

Comment:
Brigade Metrics, for now, is a separate project and basically a bolt-on. Does that influence whether we should or shouldn't cover it here in these docs? It's not strongly held, but I think my opinion is that it's only worthy of a brief mention in a section on other things in the extended Brigade 2 ecosystem.

Reply:
Fair point; removed for now.